wintaser/MainMenu.cpp

```diff
@@ -5,15 +5,23 @@
 #include "ipc.h"
 using namespace Config;
 
-#define MENU_L(smenu, pos, flags, id, suffixe, str, disableReason)	do{									\
-/*GetPrivateProfileStringA(language_name[Language], (str), (def), Str_Tmp, 1024, Language_Path);*/	\
-if((disableReason) && ((flags) & (MF_DISABLED|MF_GRAYED))) \
-	sprintf(Str_Tmp, "%s (%s)", str, disableReason); \
-else \
-	strcpy(Str_Tmp, str); \
-strcat(Str_Tmp, (suffixe)); /*AddHotkeySuffix(Str_Tmp, id, suffixe, true);*/							\
-InsertMenu((smenu), (pos), (flags), (id), Str_Tmp); } while(0)
+void MENU_L(HMENU smenu, UINT pos, UINT flags, UINT_PTR id, LPCSTR suffixe, LPCSTR str, LPCSTR disableReason)
+{
+	char Str_Tmp[1024];
+	/*GetPrivateProfileStringA(language_name[Language], str, def, Str_Tmp, 1024, Language_Path);*/
+	if (disableReason && (flags & (MF_DISABLED | MF_GRAYED)))
+		sprintf(Str_Tmp, "%s (%s)", str, disableReason);
+	else
+		strcpy(Str_Tmp, str);
+	strcat(Str_Tmp, (suffixe));
+	/*AddHotkeySuffix(Str_Tmp, id, suffixe, true);*/
+	InsertMenu(smenu, pos, flags, id, Str_Tmp);
+}
 
+__forceinline void MENU_L(HMENU smenu, UINT pos, UINT flags, HMENU menu, LPCSTR suffixe, LPCSTR str, LPCSTR disableReason)

```

Comment:
Sorry, I expressed that badly. There is no reason for those functions to be named the same. It just makes it harder to follow the code.
MENU_L is already a terrible name for the function. I'd rename them both to something like InsertMenu and InsertMenuItem.